lib/controllers/medications.js

```diff
@@ -69,7 +71,7 @@ function authorize(access) {
 }
 
 // View notification time settings
-medications.get("/:medicationid/times/:timeid", authorize("read"), function(req, res, next) {
+medications.get("/:medicationid/times/:timeid", guard(["admin", "clinician", "user"]), authorize("read"), function(req, res, next) {
```

Comment:
Why no programAdmin on this one? They can hit all the endpoints in `all_medications`.

Reply:
@B3rry @mountHouli this file will be especially important for your review

Reply:
You're right, programAdmin should probably be included on this, I'll let @B3rry @mountHouli review first.